db/migrate/20180126035928_create_categories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration[5.1]
+  def change
+    create_table :categories do |t|
+      t.text :name
+
+      t.timestamps
```

Comment:
Do we need timestamps for categories?

Reply:
We made timestamps for genres in film files, so I don't think it would be wrong necessarily to have it for categories